### ISSUE TITLE
[Analytics] Test Product Health Data with Data Stakeholders

### ISSUE THREAD

**Issue by joanneesteban** [department-of-veterans-affairs/va.gov-team#996]
## User Story
As a product health tool user, I need available analytics tools on the product health tool to show accurate data so I can understand what the data on the final product could look like.

## Goal
Testing Grey Sky data with data stakeholders.

## Acceptance Criteria
- [x] Jeff and Amy meet to verify Foresee data
- [ ] _Stretch_ Amy incorporates data into "grey sky" product health tool
- [x] Taylor Skinner and Amy meet to verify data
- [ ] _Stretch_ Amy incorporates data into "grey sky" product health tool
- [x] Nedie and Amy meet to verify data
- [ ] _Stretch_ Amy incorporates data into "grey sky" product health tool

## Definition of Done
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by joanneesteban** [comment]
@amycesal Are these getting scheduled? Let me know if you need assistance with who the SMEs are and scheduling these meetings.

**Comment by amycesal** [comment]
I was waiting to have a research plan and a grey sky prototype to actually schedule these. 

Although creating a list of contacts would be great.
- Chris
- Leah
- Rachel
- Andrea
- Matt?
- Kevin?

**Comment by joanneesteban** [comment]
@amycesal I believe this issue was to validate the data that could be used on the prototype, which should be able to happen concurrently. So that would be:
Chante - Call Center
Taylor Skinner (& TJ) - Grafana
Jeff Barnes - Foresee
Nedie - Google Analytics

And then users/stakeholders would be:
DSVA: Leah, Chris, Patrick
Leadership: Rachael, Andrew G, Emily
Product:
- Andrea Schneider
- Matt
- Kevin
- Kara
- Justin 
- Yana
- Dorothy
- Mark G

**Comment by amycesal** [comment]
### Notes from meeting with @JeffBarnesUSDS 

- Insights and Analytics team has access to Forsee Data
- Currently most data is from the intercept survey, which targets all of VA.gov, not individual products
- Starting to use the feedback survey on certain pages, but there is no historical data, and there will be fewer responses than the intercept survey because it doesn't intercept/pop-up
- could turn on something in forsee to get more analysis around the text in the "other" category
- add a satisfaction/time variable to the overall score

Basically the data that's included is possible to obtain, however we would have to change how we currently do forsee surveys and there would not be any historical data.

**Comment by amycesal** [comment]
### Notes from meeting with @clantosswett 

- Not possible to do get sentiment of calls
- currently the only data source possible is the White House Hotline data which is in Salesforce
- don't currently do a "how was this call resolved" post-call survey, but it might happen in the future
- Other call centers might be possible to get data from in the future
- Harder but not impossible to get data about certain products. Would need to change the current data collection scheme.

**Comment by amycesal** [comment]
### Notes from meeting with @ricetj @tskinn 

- Biggest question: defining uptime or downtime
- One factor is that one service is down like 30% of the time
- Which services affect which products will take some work. Not something that’s done already. Would take work to do. 
- Open question: is it easier to start with services and map to products or start with products and map which services affect them
- Not actively monitoring= qualify as up until we get an error that something bounces that it’s down.

**Comment by amycesal** [comment]
### Notes from meeting with @nedierecel @jonwehausen 
- Time to complete - don’t have but could get. Could show distribution
- Users and submissions - source from 2 different sets of data because of vets.gov
- Time of day, possible but not a thing we have currently implemented. Currently just track the day.

### Data that doesn't fall into anyone's current domain 
- Manual tracking of some things like like number of pages
- Product owner to figure out # of questions and numbers of clicks
- PA&I - processing time and time to receive benefits (product owners and Joanne?) 
- Paper form submissions? Who owns this? Product owners?
